grid2op/utils/l2rpn_idf_2023_scores.py

```diff
@@ -103,32 +104,41 @@ def __init__(
             nb_process_stats=nb_process_stats,
             scores_func={
                 "grid_operational_cost": L2RPNSandBoxScore,
+                "assistant_confidence": _AlertTrustScore,
+                # "assistant_cost": _AlertCostScore,
                 "new_renewable_sources_usage": _NewRenewableSourcesUsageScore,
             },
             score_names=["grid_operational_cost_scores",
+                         "assistant_confidence_scores",
+                         # "assistant_cost_scores",
                          "new_renewable_sources_usage_scores"],
             add_nb_highres_sim=add_nb_highres_sim,
         )
-        
-        assert(weight_op_score + weight_assistant_score + weight_nres_score==1.)
-        assert(all([weight_confidence_assistant_score>=0., weight_confidence_assistant_score<=1.]))
-        
+
+        test_weights = weight_op_score + weight_assistant_score + weight_nres_score
+        if test_weights != 1.0:
+            raise Grid2OpException(
+                'The weights of each component of the score shall sum to 1'
+            )
+        # assert(all([weight_confidence_assistant_score>=0., weight_confidence_assistant_score<=1.]))
+
         self.scale_assistant_score = scale_assistant_score
         self.scale_nres_score = scale_nres_score
         self.weight_op_score = weight_op_score
         self.weight_assistant_score = weight_assistant_score
         self.weight_nres_score = weight_nres_score
-        self.weight_confidence_assistant_score = weight_confidence_assistant_score
+        # self.weight_confidence_assistant_score = weight_confidence_assistant_score
```

Comment:
This has been commented, we should maybe juste remove it (we no longer have two components in the assistant score).